src/olympia/lib/crypto/packaged.py

```diff
@@ -62,10 +62,7 @@ def call_signing(file_obj, endpoint):
         temp_filename = temp_file.name
```

Comment:
Can you move that below, closer to where it's needed ?

Reply:
Done in fd01519